viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -550,20 +534,17 @@ export class Cognite3DViewer {
       | DisposedDelegate
   ): void {
     switch (event) {
-      case 'click':
-        this._events.click.subscribe(callback as PointerEventDelegate);
-        break;
-
       case 'hover':
```

Comment:
`hover` event is needed for `360 Image Icons` and as `hover` does not have `break` statement, it will be triggering `click` callback